db/migrations/20230227082006_users.sql

```diff
@@ -0,0 +1,16 @@
+-- migrate:up
+CREATE TABLE users(
+  id INT NOT NULL AUTO_INCREMENT,
+  kakao_id BIGINT(20) UNIQUE KEY NULL,
```

Comment:
BIGINT 자체는 데이터 크기가 정해져 있으므로 아래와 같이 수정해 주세요.
```suggestion
  kakao_id BIGINT UNIQUE KEY NULL,
```